### PR TITLE
add support for chef 13 and ubuntu 16.04

### PR DESCRIPTION
**Pull request by davidsayers**

### CODE REVIEW

**Comment by roboshopkeep** [comment]
Specs Fail :-1:

**Comment by roboshopkeep** [comment]
Specs Fail :-1:

**Comment by davidsayers** [comment]
Specs!

**Comment by roboshopkeep** [comment]
Specs Fail :-1:

**Comment by roboshopkeep** [comment]
Specs Fail :-1:

**Comment by davidsayers** [comment]
Specs!

**Comment by roboshopkeep** [comment]
Specs Fail :-1:

**Comment by roboshopkeep** [comment]
Specs Pass! :+1:

File: recipes/_install.rb [hunk 1]
```diff
@@ -31,14 +33,14 @@
   source node['looker']['startup_script_url']
   owner 'looker'
   group 'looker'
-  mode 0750
+  mode 0o750
```
Comment (darrylb-github) [inline recipes/_install.rb]: @davidsayers What does this format do vs just `0750` out of interest?

**Comment by darrylb-github** [comment]
LFTM 👍 

Added 1 question above for my own interest.

File: recipes/_install.rb (hunk 1 above)
Comment (davidsayers) [reply recipes/_install.rb]: ummm, rubocop did it :-)

Think it's an octal thing.

**Pull request merged**